JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.csv

```diff
@@ -118,12 +118,12 @@ representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,,A relationshi
 representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,exactly 1,,https://www.theworldavatar.com/kg/ontobuiltenv/,representative for
 Bank,Class,Non-Domestic,IS-A,,,,A property where customers can deposit and withdraw money.,https://www.theworldavatar.com/kg/ontobuiltenv/,Bank
 Pharmacy,Class,Medical,IS-A,,,,A healthcare facility where medicinal drugs are prepared or sold.,https://www.theworldavatar.com/kg/ontobuiltenv/,Pharmacy
-BuildingFloors,Class,,,Property,,,A concept to describe amount of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Building floors
-hasBuildingFloors,Object Property,,,Property,BuildingFloors,exactly 1,A relationship between a property and its amount of floors.,https://www.theworldavatar.com/kg/ontobuiltenv/,has building floors
-hasFloorsValue,Data Property,,,BuildingFloors,integer,exactly 1,A relationship between the floors and its value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has floors value
-FloorsCatA,Class,BuildingFloors,IS-A,,,,Category A of floors: data from reliable departments (e.g. government agencies),https://www.theworldavatar.com/kg/ontobuiltenv/,floors category A
-FloorsCatB,Class,BuildingFloors,IS-A,,,,Category B of floors: open source data without any guarantees (e.g. OSM),https://www.theworldavatar.com/kg/ontobuiltenv/,floors category B
-FloorsCatC,Class,BuildingFloors,IS-A,,,,Category C of floors: estimate,https://www.theworldavatar.com/kg/ontobuiltenv/,floors category C
+hasNumberOfFloors,Object Property,,,Property,NumberOfFloors,exactly 1,A relationship between a property and its number of floors.,https://www.theworldavatar.com/kg/ontobuiltenv/,has number of floors
+NumberOfFloors,Class,,,Property,,,A concept to describe number of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Number of floors
+hasNumberOfFloorsValue,Data Property,,,NumberOfFloors,integer,exactly 1,A relationship between the number of floors and its numerical value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has number of floors value
```

Comment:
Given that the domain class is now called `NumberOfFloors`, it would be customary to name this data property simply `hasValue`. (This is analogous to any quantity that e.g. has a value and a unit. The meaning is now obvious, given the new name of the domain class.)

Reply:
ok I will change it to `hasValue`. I was thinking to use `hasValue`, just afraid it's too simply and confuse with the other property definition. As you said, it seems no issue.